src/identity/components/MarketoAccountUpgradeOverlay.scss

```diff
@@ -0,0 +1,12 @@
+@import '@influxdata/clockface/dist/variables.scss';
+
+#mktoForm_2826 {
+  display: none;
+}
+
```

Comment:
The marketo form styling isn't pretty and is inconsistent with the app. To resolve that, we use our own submit button, and hide the marketo form itself.